src/components/UpLinks/UpLinks.test.tsx

```diff
@@ -6,9 +6,9 @@ import { DetailContext } from '../../pages/Version';
 import UpLinks from './UpLinks';
 
 describe('<UpLinks /> component', () => {
-  test('should render the component in default state', () => {
+  test('should return null without packageMeta', () => {
     const wrapper = shallow(<UpLinks />);
-    expect(wrapper.html()).toMatchSnapshot();
+    expect(wrapper.html()).toBeNull();
```

Comment:
Hmmm here you really removed the snapshot...I would rather leave it here, because even if  snapshots don't seem to be efficient, I've already caught many things through them, avoiding possible errors.

Reply:
@priscilawebdev I've checked this snapshot before removing it, and it was only a `null` that was been saved. Because of this, I decided to remove it and leave only the assertion, since a `null` snapshot does not bring so much value and it's easier for a developer to ignore it.

Reply:
@hugoazevedosoares it makes sense! I agree :-) thank you!